LICENSE

```diff
@@ -1,9 +1,6 @@
-Copyright (c) 2015 Uber Technologies, Inc.
-
-This software includes parts of PhiloGL (https://github.com/philogb/philogl)
-under MIT license. PhiloGL parts Copyright © 2013 Sencha Labs.
+loaders.gl is licensed under the MIT license
 
-This software includes adaptations of postprocessing code from THREE.js (https://github.com/mrdoob/three.js/) under MIT license. Additional attribution given in specific source files. THREE.js parts Copyright © 2010-2018 three.js authors.
+Copyright (c) vis.gl contributors
```

Comment:
Formatting suggested by the [Linux foundation](https://spdx.org/licenses/MIT.html):
```
MIT License

Copyright (c) vis.gl contributors
Copyright (c) 2015 Uber Technologies, Inc.

Permission is hereby granted...
```
The `loaders.gl is licensed under the MIT license` phrasing is probably fine though since it's specifying among multiple licenses.